testcases/kernel/syscalls/getsockopt/getsockopt01.c

```diff
@@ -1,208 +1,88 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
 /*
- *
- *   Copyright (c) International Business Machines  Corp., 2001
- *
- *   This program is free software;  you can redistribute it and/or modify
- *   it under the terms of the GNU General Public License as published by
- *   the Free Software Foundation; either version 2 of the License, or
- *   (at your option) any later version.
- *
- *   This program is distributed in the hope that it will be useful,
- *   but WITHOUT ANY WARRANTY;  without even the implied warranty of
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
- *   the GNU General Public License for more details.
- *
- *   You should have received a copy of the GNU General Public License
- *   along with this program;  if not, write to the Free Software
- *   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ * Copyright (c) International Business Machines  Corp., 2001
+ * 07/2001 Ported by Wayne Boyer
+ * Copyright (C) 2024 SUSE LLC Andrea Manzini <andrea.manzini@suse.com>
  */
 
-/*
- * Test Name: getsockopt01
- *
- * Test Description:
- *  Verify that getsockopt() returns the proper errno for various failure cases
- *
- * Usage:  <for command-line>
- *  getsockopt01 [-c n] [-e] [-i n] [-I x] [-P x] [-t]
- *     where,  -c n : Run n copies concurrently.
- *             -e   : Turn on errno logging.
- *	       -i n : Execute test n times.
- *	       -I x : Execute test for x seconds.
- *	       -P x : Pause for x seconds between iterations.
- *	       -t   : Turn on syscall timing.
- *
- * HISTORY
- *	07/2001 Ported by Wayne Boyer
- *
- * RESTRICTIONS:
- *  None.
+/*\
+ * [Description]
+ * Verify that getsockopt() returns the proper errno for various failure cases:
```

Comment:
Here needs to be a blank line, otherwise the following will not be list. NOTE: I might send v2 for approach which would fix it (https://patchwork.ozlabs.org/project/ltp/list/?series=388930&state=*, https://lore.kernel.org/ltp/20240104204614.1426027-1-pvorel@suse.cz/), but until fix merged it's needed to remember this).